site/content/docs/user/quick-start.md

```diff
@@ -363,6 +363,30 @@ The structure of the logs will look more or less like this:
 The logs contain information about the Docker host, the containers running 
 kind, the Kubernetes cluster itself, etc.
 
+### Configuring kubelet Garbage Collection in Your Cluster
```

Comment:
I can see documenting this being useful, but this is pretty far into "quick-start has way too much in it"

Reply:
yes. I have the same opinion as @aojea . https://github.com/kubernetes-sigs/kind/pull/572#discussion_r290599547

> we should first thing how do we want to reorganize the docs, seems that they are starting to grow and I think that adding an advanced section with this stuff is not a good solution because we are going to have the same problem after some time.

Reply:
in addition to this PR, I see that we have some documentation on how to use kind in CI or some extra advanced features, which may need to be reorganized.

Reply:
kind in CI is now in the resources doc

I also don't think this is safe to enable in kind. it can evict the core images.